meta-facebook/yv35-cl/src/lib/plat_spi.h

```diff
@@ -0,0 +1,4 @@
+#ifndef PLAT_SPI_H
+#define PLAT_SPI_H
+
+#endif
```

Comment:
Is this file supposed to not contain anything?

Reply:
We have removed this file and done the forced update.
Please review it. Thanks.